docs/docs/shapes/fitbox.md

```diff
@@ -0,0 +1,46 @@
+---
+id: fitbox
+title: FitBox
+sidebar_label: FitBox
+slug: /shapes/fibox
+---
+
+The `FitBox` components allows you to automatically scales drawings to you Canvas.
```

Comment:
**Suggestion:**
The `FitBox` components allows you to automatically scale drawings so they fit into a destination rectangle.

Reply:
done!